### PR TITLE
privacy-policy

### PR DESCRIPTION
**Pull request by freedannyv**
#393 
# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x ] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

Created new Vue (PrivacyPolicy.vue) for Privacy Policy text. 
Created new path in router/CustomerView.vue for the Privacy Policy.
 

# How to Test/Reproduce

Provide steps on how one can test your changes here. e.g.
1. Log in as '...'
2. Go to 'www.foodouken.com/privacy-policy' to view page.
3. Click on '...'
4. See the updated element x

# Screenshots/casts

If applicable, include any relevant screenshot or screencasts of your changes.

# Additional Context

Provide any additional notes that you think would help effective review of your code.

### CODE REVIEW

File: src/views/PrivacyPolicy.vue [hunk 1]
```diff
@@ -0,0 +1,787 @@
...
+          Meredith
+          <br />
+          meredith@whynot.earth
+          <br />
+          meredith-core, Inc. 908 Islander Way, Chattanooga, TN, 37402 USA,
+          Attention: Privacy
+        </p>
+      </div>
+    </div>
+    <hr class="my-8 border-gray-700" />
```
Comment (paulchrisluke) [inline src/views/PrivacyPolicy.vue:671]: This page still displays the home page tenants, please remove this content

File: src/views/PrivacyPolicy.vue [hunk 2]
```diff
@@ -0,0 +1,787 @@
...
+          the "unsubscribe" link located on the bottom of our e-mails, updating
+          your communication preferences, or by sending us email us at
+          meredith@whynot.earth or postal mail to meredith-core, 908 Islander
+          Way, Chattanooga, TN, 37402 USA, Attention: Privacy. Customers cannot
+          opt out of receiving transactional emails related to their account
+          with us or the Subscription Service.
+        </p>
+        <br />
+        <p>
+          **To Unsubscribe from Our Customers' Communications**
```
Comment (paulchrisluke) [inline src/views/PrivacyPolicy.vue:642]: lots of these still need to be bolded and formatting cleaned up

**Review by paulchrisluke — CHANGES_REQUESTED** [review]
couple comments, but looking great!

**Review by giteshk11 — CHANGES_REQUESTED** [review]
When I click on terms and conditions its doesn't redirect to /privacy-policy
You have added the redirection when I click on terms and conditions. @freedannyv

**Pull request closed without merging**